docs/getting-started/operating-systems.md

```diff
@@ -0,0 +1,59 @@
+# Operating System Compatibility and Differences 
+
+This page was last updated for v0.8.2
+
+---
+
+The purpose of this page is describe which binaries are available for what operating systems. It will also cover the key differences for running the cross-platform command-line applications.
+
+---
+
+## The Applications
+
+**Paymetheus**: The only *complete* GUI client as of v0.8.2. <br />
```

Comment:
I think it is more important that this is the windows GUI rather than 'only complete' gui.